build.rs

```diff
@@ -357,10 +357,19 @@ fn blob_backends(conf: &Config, build_dir: &std::path::PathBuf) -> (Vec<String>,
     if conf.use_lib {
         let afpath  = match env::var("AF_PATH") {
             Ok(af_path) => PathBuf::from(&af_path),
-            Err(_)      => panic!("Error use_lib is defined, but AF_PATH is not defined"),
+            Err(_)      => {
+                println!("WARNING! USE_LIB IS DEFINED, BUT AF_PATH IS NOT DEFINED, TRYING TO FIND LIBRARIES FROM KNOWN DEFAULT LOCATIONS");
+                PathBuf::from("/usr/")
+            },
         };
         let libpath = afpath.join("lib");
```

Comment:
`afpath` will never be `None`. Under the Err it is being set to another path location. I.e. error is handled when `AF_PATH` is not set.